community/kernel/src/main/java/org/neo4j/kernel/impl/api/index/IndexPopulationProgress.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2002-2015 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.kernel.impl.api.index;
+
+/**
+ * This class is simply a progress counter of indexing population progress. It has the constraint that
+ * 0 <= completed <= total
+ * <p>
+ * Use IndexPopulationProgress.NONE if you need an object without any particular progress.
+ */
+public class IndexPopulationProgress
```

Comment:
I would have liked a `toString` method in here which printed something like `45%`